dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -833,6 +834,34 @@ public void testGitlab() {
 
     }
 
+    /**
+     * This tests manually publishing a Bitbucket workflow
+     */
+    @Test
+    @Category(SlowTest.class)
+    public void testManualPublishBitbucket() {
+        // Setup DB
+        final CommonTestUtilities.TestingPostgres testingPostgres = getTestingPostgres();
+
+        // manual publish
+        Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file2.txt"), "workflow", "manual_publish",
+                "--repository", "dockstore-workflow", "--organization", "dockstore_testuser2", "--git-version-control", "bitbucket",
+                "--workflow-name", "testname", "--workflow-path", "/Dockstore.wdl", "--descriptor-type", "wdl", "--script" });
+
+        // Check for two valid versions (wdl_import and surprisingly, cwl_import)
+        final long count = testingPostgres
```

Comment:
this will break if someone adds a version to test for something new. 
A check for two versions that have specific names for example will tolerate a new version being added